### ISSUE TITLE
Typeorm support

### ISSUE THREAD

**Issue by njsubedi** [SoftwareBrothers/adminjs-nestjs#24]
Is there a plan to support @nestjs/typeorm in this plugin?

**Comment by AlmogCohen** [comment]
Yup, I was trying to make it work with typeorm and it's a very hard process. Couldn't make it work chasing one bug after the other.

**Comment by dziraf** [comment]
Our example app repository will soon be updated with more usage examples. Currently you can see NestJS examples here:
https://github.com/SoftwareBrothers/adminjs-example-app/tree/typescript-version/src/servers/nestjs

**Comment by saratscheff** [comment]
The updated link with examples is here: https://github.com/SoftwareBrothers/adminjs-example-app/tree/master/src/servers/nestjs.

Instead of the examples though, I would suggest following the updated documentation: https://docs.adminjs.co/installation/plugins/nest

I got a production-ready version working without issues. I think this issue can be closed.
